backend/test/services.terminals.spec.js

```diff
@@ -805,10 +825,49 @@ describe('services', function () {
         terminalStub.mockReturnValue(createTerminalConfig({ gardenTerminalHost, bootstrap }))
         const bootstrapper = new Bootstrapper()
         bootstrapper.bootstrapResource(seed)
-        await pEvent(bootstrapper, 'empty')
+        await pEvent(bootstrapper, 'drain')
         const stats = bootstrapper.getStats()
         expect(stats.total).toBe(1)
         expect(stats.successRate).toBe(1)
+        expect(bootstrapper.bootstrapState.getValueForResource(seed).state).toBe(BootstrapStatusEnum.BOOTSTRAPPED)
+      })
+
+      it('should bootstrap a seed cluster after revision changed', async function () {
+        const gardenTerminalHost = {
+          seedRef: seedName
+        }
+        const bootstrap = {
+          disabled: false,
+          shootDisabled: false,
+          seedDisabled: false
+        }
+        terminalStub.mockReturnValue(createTerminalConfig({ gardenTerminalHost, bootstrap }))
+        const seed = _.find(seedList, ['metadata.name', seedName])
+        _.set(seed, 'metadata.annotations["seed.gardener.cloud/ingress-class"]', 'foo')
+        const bootstrapper = new Bootstrapper()
```

Comment:
move `bootstrapper = new Bootstrapper()` to beforeEach and define helper function for mapping the state